Dockerfile.ci

```diff
@@ -1,74 +1,7 @@
 FROM dependabot/dependabot-core
-ARG CODE_DIR=/home/dependabot/dependabot-core
-WORKDIR ${CODE_DIR}
-
-ENV BUNDLE_PATH="/home/dependabot/.bundle" \
-  BUNDLE_BIN=".bundle/binstubs" \
-  PATH=".bundle/binstubs:$PATH:/home/dependabot/.bundle/bin"
 
 RUN pyenv install -s 3.6.13
 RUN pyenv install -s 3.7.1
 RUN pyenv install -s 3.7.10
```

Comment:
Unrelated to this PR, but I noticed that these still aren't being cached currently. Do you have any idea why that could be?

Reply:
Not sure, are they being installed again in the tests?

Reply:
First hunch was some permission issue but I think these, as the tests, are running as root so should be fine.

Reply:
No, they are being installed in the docker-ci build step in the action actually, when IMO all these layers should be cached?

Reply:
Oh wait you mean on this PRs build? I busted all the docker images used here so everything needs to rebuild.

Reply:
Ah yeah I see, I wonder if the change in this PR to use the inline buildkit cache would help with this?

Reply:
Curious to find out! It might 🤞